feature/big_files_cli_spec.rb

```diff
@@ -13,7 +13,9 @@
              "\n" \
              '    -e, --exclude-glob glob here     ' \
              "Files to exclude - default is none\n" \
-             "    -h, --help                       This message\n")
+             "    -h, --help                       This message\n" \
+             '    -n, --num-files number-here      ' \
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.